packages/terra-folder-tree/src/FolderTreeUtils.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-param-reassign */
+const handleMoveFocus = (elementA, elementB) => {
```

Comment:
Can we add a comment to document the purpose/usage of this function, including what `elementA` and `elementB` are respectively?

Reply:
Updated [6c7a037](https://github.com/cerner/terra-framework/pull/1967/commits/6c7a03711ef1d99939b2d8222d8eacee3aa61d63)